Improve stat printer

sknorr opened the pull request:
Now a separate function. Does not tell you if it encountered 0 errors (i.e. it now gives you relevant messages only).

Not quite sure about placing the function in display.py -- input welcome.
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/openSUSE/docmanager/pull/114%23issuecomment-229926642%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/openSUSE/docmanager/pull/114%23issuecomment-229926642%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Got%20oral%20approval%20from%20toms...%20So%20merging.%22%2C%20%22created_at%22%3A%20%222016-07-01T11%3A45%3A10Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/5476547%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/sknorr%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'><img src='http://www.codereviewhub.com/site/github-bar.png' height=40></a>
- [ ] <a href='#crh-comment-General Comment'></a> <img src='http://www.codereviewhub.com/site/github-remaining.png' height=16 width=60>&nbsp;<b><a href='https://github.com/openSUSE/docmanager/pull/114#issuecomment-229926642'>General Comment</a></b>
- <a href='https://github.com/sknorr'><img border=0 src='https://avatars.githubusercontent.com/u/5476547?v=3' height=16 width=16'></a> Got oral approval from toms... So merging.

<a href='https://www.codereviewhub.com/openSUSE/docmanager/pull/114?mark_as_completed=1'><img src='http://www.codereviewhub.com/site/github-mark-as-completed.png' height=26></a>&nbsp;<a href='https://www.codereviewhub.com/openSUSE/docmanager/pull/114?approve=1'><img src='http://www.codereviewhub.com/site/github-approve.png' height=26></a>&nbsp;<a href='https://github.com/openSUSE/docmanager/pull/114'><img src='http://www.codereviewhub.com/site/github-refresh.png' height=26></a>
<a href='#crh-end'></a>

Commit:
Improve stat printer

+ separate function
+ does not tell you if it encountered 0 errors

tomschr commented:
This pull request is now managed with CodeReviewHub. <a href='https://github.com/openSUSE/docmanager/pull/114'>Refresh</a>.<a href='#crh-update-none'></a>

Commit:
Improve over stats messages as well

No point in making a function here, but still could avoid these from
being so talkative.

Commit:
Improve log message a little

tomschr committed:
Rename makefile -> Makefile

tomschr committed:
Move pytest section into setup.cfg

tomschr committed:
Add more entries suggested by check-manifest

tomschr committed:
Completely revamp tox.ini

* add matrix

tomschr committed:
Replace coverage with tox

tomschr committed:
Improve .travis.yml

* split responsibilites: tox and all unit testing, coverage is done
  in .travis.yml, whereas docmanager acceptance testing is done
  with .travis-script.sh

tomschr committed:
Use docmanager arguments: * -> *.xml

tomschr committed:
Improve .travis.yml

* require sudo
* disable 3.5 and 3.6 for the time being :-(
* install xsltproc docbook-xsl

ghost commented:
Got oral approval from toms... So merging.
